bundledata.go

```diff
@@ -141,6 +141,9 @@ type ServiceSpec struct {
 	// Storage holds the constraints for storage to assign
 	// to units of the service.
 	Storage map[string]string `bson:",omitempty" json:",omitempty" yaml:",omitempty"`
+
+	// EndpointBindings maps how endpoints are bound to spaces
+	EndpointBindings map[string]string `bson:"bindings,omitempty" json:"bindings,omitempty" yaml:"bindings,omitempty"`
```

Comment:
We want the YAML field to be call "bindings" but, internally, we always have used EndpointBindings, hence the explicit name.